ICE on crate not found

<!--
Thank you for finding an Internal Compiler Error! 🧊  If possible, try to provide
a minimal verifiable example. You can read "Rust Bug Minimization Patterns" for
how to create smaller examples.

http://blog.pnkfx.org/blog/2019/11/18/rust-bug-minimization-patterns/

-->

### Code

```rust
extern crate test_macro;
```
With the following launch option:

```
./gcc/crab1 -frust-name-resolution-2.0 ./test.rs
```
Extern crate option has been omitted on purpose.

### Meta

4d6a0094e9b

### Error output

```
./test.rs:1:1: error: failed to locate crate ‘test_macro’
    1 | extern crate test_macro;
      | ^~~~~~
```


<details><summary>Backtrace</summary>
  <p>
  
  ```
crab1: internal compiler error: in visit, at rust/resolve/rust-toplevel-name-resolver-2.0.cc:97
0x1411b2b Rust::Resolver2_0::TopLevel::visit(Rust::AST::ExternCrate&)
	/home/patryp/Documents/gccrs_bis/build/gcc/../../gcc/rust/resolve/rust-toplevel-name-resolver-2.0.cc:96
0x1138f30 Rust::AST::ExternCrate::accept_vis(Rust::AST::ASTVisitor&)
	/home/patryp/Documents/gccrs_bis/build/gcc/../../gcc/rust/ast/rust-ast.cc:4738
0x141193a Rust::Resolver2_0::TopLevel::go(Rust::AST::Crate&)
	/home/patryp/Documents/gccrs_bis/build/gcc/../../gcc/rust/resolve/rust-toplevel-name-resolver-2.0.cc:58
0x1418704 Rust::Resolver2_0::Early::go(Rust::AST::Crate&)
	/home/patryp/Documents/gccrs_bis/build/gcc/../../gcc/rust/resolve/rust-early-name-resolver-2.0.cc:34
0x1262e68 Rust::Session::expansion(Rust::AST::Crate&)
	/home/patryp/Documents/gccrs_bis/build/gcc/../../gcc/rust/rust-session-manager.cc:886
0x12619d5 Rust::Session::compile_crate(char const*)
	/home/patryp/Documents/gccrs_bis/build/gcc/../../gcc/rust/rust-session-manager.cc:588
0x1261424 Rust::Session::handle_input_files(int, char const**)
	/home/patryp/Documents/gccrs_bis/build/gcc/../../gcc/rust/rust-session-manager.cc:402
0x10d6142 grs_langhook_parse_file()
	/home/patryp/Documents/gccrs_bis/build/gcc/../../gcc/rust/rust-lang.cc:165
Please submit a full bug report, with preprocessed source (by using -freport-bug).
Please include the complete backtrace with any bug report.
See <https://gcc.gnu.org/bugs/> for instructions.
  ```
  
  </p>
</details>

--- a/gcc/rust/rust-session-manager.cc
+++ b/gcc/rust/rust-session-manager.cc
@@ -877,6 +877,9 @@ Session::expansion (AST::Crate &crate)
   while (!fixed_point_reached && iterations < cfg.recursion_limit)
     {
       CfgStrip ().go (crate);
+      // Errors might happen during cfg strip pass
+      if (saw_errors ())
+	break;
 
       auto ctx = Resolver2_0::NameResolutionContext ();
 

